haskell-indexer-backend-ghc/tests/Language/Haskell/Indexer/Backend/Ghc/Test/BasicTestBase.hs

```diff
@@ -393,6 +393,15 @@ testImportRefsHiding =
           includesPos (11, 1) u3 -- type signature in ImpExpDefs.hs
         us -> checking $ assertFailure "Usage count differs for bar"
 
+testPatternSynonyms :: AssertionInEnv
+testPatternSynonyms = assertXRefsFrom ["basic/PatSyn.hs"] $ do
+    declAt (4, 9) >>= singleUsage >>= includesPos (6, 15)
+    declAt (8, 9) >>= usages >>= \case
+      [u1, u2] -> do
+        includesPos (10, 14) u1
+        includesPos (12, 14) u2
+      _ -> checking $ assertFailure "Usage count differs for BiSingle"
```

Comment:
Maybe include the value into the error message?

Reply:
This recurs quite some time, so either one would need to generalize the pattern used here, or live with "debugging when it fails".

Reply:
Update all the places in this file.